cli/src/main/java/org/apache/iotdb/tool/ImportCsv.java

```diff
@@ -65,14 +72,22 @@
   private static final String FAILED_FILE_ARGS = "fd";
   private static final String FAILED_FILE_NAME = "failed file directory";
 
+  private static final String ALIGNED_ARGS = "aligned";
+  private static final String ALIGNED_NAME = "use the aligned interface";
+
```

Comment:
This is for aligned timeseries. We shouldn't introduce it to 0.12.

Reply:
got it!